src/lib.rs

```diff
@@ -389,6 +389,11 @@ fn create_settings(matches: &ArgMatches) -> Settings {
     });
 
     settings.argv = argv;
+    settings.orig_argv = vec![env::current_exe()
+        .unwrap_or(PathBuf::new())
+        .to_str()
+        .unwrap()
+        .to_string()];
```

Comment:
The code looks putting only executable names not the full argv.
The value also need to be fetched from actual argv rather than 
directly processing executable name.

```python
import sys

print(sys.orig_argv)
```

```
$ python f.py -a -b -c
['~/.pyenv/versions/RustPython/bin/python', 'f.py', '-a', '-b', '-c']
```

Reply:
I fixed it to get the correct result, but still don't know how to get the executable name indirectly...